stdlib/src/builtin/simd.mojo

```diff
@@ -2151,6 +2151,30 @@ struct SIMD[type: DType, size: Int](
         constrained[type.is_numeric(), "the SIMD type must be numeric"]()
         return __mlir_op.`pop.max`(self.value, other.value)
 
+    fn count(self, value: Scalar[type]) -> Int:
+        """Count the occurences the value in the vector.
+
+        Example usage:
+
+        ```mojo
+        var thirty_two_uint8 = SIMD[DType.uint8, 32](1)
+        thirty_two_uint8[0] = 0
+        print(thirty_two_uint8.count(1)) # 31
```

Comment:
```suggestion
        print(SIMD[DType.uint8, 4](1, 1, 1, 0).count(1)) # 3
```

Reply:
Seems that this was resolved, but not introduced

Reply:
We got this one :+1: